### ISSUE TITLE
[Feature Request] Add move window to left/right desktop accordingly.

### ISSUE THREAD

**Issue by bigplayer-ai** [newlooper/VirtualSpace#46]
Hey,
Thanks for adding moving specific windows to specific desktops.
I wish You can add moving windows to left/right/up/down desktops accordingly. This way the UI/user workflow would be more consistent.
![Clip_20230301_133942](https://user-images.githubusercontent.com/119108387/222129390-0be8cffd-2790-494e-82c3-306aeae9e2d5.png)

**Comment by akoyich** [comment]
This is the same as this, right?

https://github.com/newlooper/VirtualSpace/issues/60

**Comment by kranix0** [comment]
> This is the same as this, right?
> 
> #60

No, this one is about moving between virtual desktops. #60 is about moving between different _displays_ (monitors).
